keylime/mba/elparsing/tpm_bootlog_enrich.py

```diff
@@ -23,7 +23,7 @@
 ##################################################################################
 
 
-class hexint(int):
```

Comment:
I believe this class name is indeed invalid as it should start with a capital letter.
You could fix this as part of this PR, but I'm OK if you prefer to leave this to a future PR